hardware/ITGWBase.cpp

```diff
@@ -15,7 +15,7 @@
 #include "itgw/ITGWSwitchIntertechno.h"
 #include "itgw/ITGWSwitchBrennenstuhl.h"
 
-
+#define _DEBUG
```

Comment:
No, it's not the way, that's why I forgot about it, but it works for the purpose i needed it. I do my day job on windows. I prefer my free time on Linux. :) Just because I can. I'm told codeblocks is good for c++ but I'm used to netbeans and I couldn't be bothered to find out why the debugging wasn't working the way it was supposed to just to get debud ifdef-ed printouts :P